### PR TITLE
Feature/reading analytics api

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[會員制] 閱讀文章篇數、閱讀時間串接](https://app.asana.com/0/1203699264568808/1206008027632075/f)

# Notice

# Dependency

### CODE REVIEW

File: packages/redux/src/actions/__test__/analytics.test.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            payload: {
+              statusCode: 200,
+              data: mockSetUserAnalyticsDataResponse,
+            },
+          }
+          expect(store.getActions()[0]).toEqual(requestExp)
+          expect(store.getActions()[1]).toEqual(successExp)
+        })
+    })
+    test('should dispatch an action created by `setUserAnalyticsData` if updating failed', () => {
```
Comment (Aylie-Chou) [inline packages/redux/src/actions/__test__/analytics.test.js:65]: Could we also add some cases for `400` & `401` errors

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
Mostly looking gooood !
As previous discussion, it would be great to also add fail test cases for `400` errors, please have a look.

File: packages/redux/src/actions/__test__/analytics.test.js (hunk 1 above)
Comment (liruchen32) [reply packages/redux/src/actions/__test__/analytics.test.js]: fixed

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**